app/components/subcases/subcases-overview-header.hbs

```diff
@@ -6,7 +6,7 @@
           @skin="borderless"
           @icon="chevron-left"
           @layout="icon-only"
-          {{on "click" @backLink}}
+          {{on "click" this.goBackToCases}}
```

Comment:
Hier zou ik een `LinkTo` component gebruiken ipv een button met een action. Er is waarschijnlijk ook een variant van de `Auk::Button` die een `@route` param of iets dergelijks ondersteunt.